config.toml.example

```diff
@@ -5,6 +5,9 @@ api_secret = "golbat"   # Golbat secret required on api calls (blank for none)
 stats = false           # Enable creation of pokemon_timing and pokemon_history table (now recommended=false)
 in_memory = false       # Use in-memory storage for pokemon only
 
+# koji_url = "http://{koji_url}/api/v1/geofence/feature-collection/{golbat_project}"
```

Comment:
Do we need a koji section? Is this likely to be it for settings?

Reply:
Added just in case.